content/en/integrations/rsyslog.md

```diff
@@ -41,16 +41,43 @@ Configure Rsyslog to gather logs from your host, containers, & services.
 
 2. Create an `/etc/rsyslog.d/datadog.conf` file.
 
+{{< site-region region="us,eu" >}}
 
-3. In `/etc/rsyslog.d/datadog.conf`, add the following configuration. You must include a separate `input` line for each log file you want to monitor:
+3. In `/etc/rsyslog.d/datadog.conf`, add the following configuration and replace `<site_url>` with **{{< region-param key="dd_site" >}}**. You must include a separate `input` line for each log file you want to monitor:
```

Comment:
I bolded the site URL because if I tried to put it in code (``), it would break the shortcode.